clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -809,7 +809,7 @@ public void run() {
 
                 // Handle new calls and metadata update requests.
                 prevMetadataVersion = checkMetadataReady(prevMetadataVersion);
-                long pollTimeout = 1200000;
+                long pollTimeout = 100;
```

Comment:
This was just a hack to demonstrate the issue, but without this change the fallback times out.